shared/settings/index.js

```diff
@@ -65,9 +65,7 @@ const Connected = compose(
 
 // $FlowIssue lets fix this
 Connected.navigationOptions = {
```

Comment:
There should still be a blank header right?
<img width="438" alt="image" src="https://user-images.githubusercontent.com/11968340/57083395-7abc5c80-6cc6-11e9-937a-d3a466fcb05c.png">

Reply:
i thought it looked better w/o this blank space. @cecileboucheron ?